ocaml/tests/test_event.ml

```diff
@@ -57,6 +57,36 @@ let test_event_from_ev_rel () =
   Alcotest.(check int) "list length" 1 (List.length vm_ev);
   Alcotest.(check int) "list length" 1 (List.length vbd_ev)
 
+let test_events_from_obj () =
+  (* Test that events on one object create the correct number of events on other objects *)
+  let __context, session_id = event_setup_common () in
+  let evs = Xapi_event.from __context ["vm"] "" 30.0 |> parse_event_from in
+  let vma = make_vm __context () in
```

Comment:
Aren't these in the wrong order? Don't you want to do a thing, _then_ get the events?